rfcs/0001-bootstrapping-rfc-process.md

```diff
@@ -0,0 +1,54 @@
+- Feature Name: bootstrapping-rfc-process
+- Start Date: 09 February 2018
+- RFC PR: (leave this empty)
+- IPFS Issue: (leave this empty)
+
+# Summary
+[summary]: #summary
+
+Begin using an RFC process. Initially use a structure similar to the process used by the maintainers of Rust lang, as described in https://github.com/rust-lang/rfcs
+
+# Motivation
+[motivation]: #motivation
+
+Establish a clear, repeatable process for people to propose ideas and make decisions in a way that
```

Comment:
I think it would be best if supporting questions were removed from RFC in its final form. They just decrease Signal to Noise ratio.

Reply:
I don't understand what you're proposing here. What supporting questions? Where in the document are you suggesting they should be removed from? At what point in the process should they be removed? Who should remove them? Why should they remove them?

Reply:
I'm saying, the questions from the RFC template should not be part of final RFC documents.

See all of rusts RFC documents. For example, motivation section does not contain the "Establish a clear, repeatable process for people to propose ideas and make decisions in a way that" from template.

One example, but it is the same for all of them:
https://github.com/rust-lang/rfcs/blob/master/text/2333-prior-art.md

Reply:
Do you have a suggestion of how to encourage people to do this? It seems to work fine with Rust using the template and RFC process as-is. Do you want to add a step to the process where someone has to check if these are removed?

Reply:
It will come up in the review, and clear up when are few RFCs already out.
From the beginning I treated it like that (I started writing few RFCs already), remove the questions after that part is written. I think we don't need to add anything to the process for it apart from making sure that all RFCs are clear of it.